### ISSUE TITLE
esphome - basestation disconnected when esp connects

### ISSUE THREAD

**Issue by TiboGabriels** [jariz/homeassistant-basestation#10]
This is more about your blog post regarding espHome where you use an esp32 to poll the basestations.
This guide was very easy to get setup and running, but the issue is that as soon as the esp connects to the basestations, my pc no longer can. This means that while i can read the base stations status in HA, when I quite steamvr it can no longer tell them to shutdown. Did you have this same issue? how did you resolve this?

**Comment by RafaelSchridi** [comment]
I've got the same issues using BLE proxy with a esp32, Thought my basestations were broken until i turned of the integration and they started to behave normally again.

Is the ESP32 secretly holding connections or failing to release them?

**Comment by systemofapwne** [comment]
OH - MY - GOD.
I was trying - desperately - to get my basestations into HA via my ESP32 BLE-Proxies. No chance. They never got added (MAC was discovered though). https://github.com/MineTech13/homeassistant-basestation/issues/1#issuecomment-2681189511

After finding your bugreport, it just made * CLICK *: I disabled Bluetooth on my phone (where I have the [Lighthouse PM](https://github.com/jeroen1602/lighthouse_pm/releases) app installed to controle the basestations).
Now, the integration *finally* was able to interact with my basestations.

**My assesment**
There seems to be a competition between "who has control".
The strange thing: 
- With that android app and bluetooth on my phone running, my Valve Index was always able to wake/sleep the basestations.
- With the app running, HA cannot interact with the basestations
- And apparently to your bugreport, the HMDs like the Valve Index cannot interact with basestations anymore, when the ESP32 BLE-Proxy is running

**Comment by TiboGabriels** [comment]
Yeah, I'm not familiar enough with Bluetooth to fix this, if it is fixable at all. If you have them in your bedroom like I doz the noise of the lighthouses means this is pretty much unusable for me.
